packages/hop-node/src/cli/unstake.ts

```diff
@@ -24,7 +61,33 @@ program
       const chain = source.chain
       const token = source.token
       const amount = Number(source.args[0] || source.amount)
-      await staker(chain, token, amount, StakerAction.Unstake)
+
+      if (!amount) {
+        throw new Error('amount is required. E.g. 100')
+      }
+      if (!chain) {
+        throw new Error('chain is required')
+      }
+
+      // Arbitrary watcher since only the bridge is needed
+      const watchers = await getWatchers({
```

Comment:
Why not just get the bridge directly here?

```typescript
import contracts from 'src/contracts'

const tokenContracts = contracts.get(token, chain)
return chain === Chain.Ethereum ? tokenContracts.l1Bridge : tokenContracts.l2Bridge
```

Reply:
It is my understanding that there is no provider associated with the bridge when we do that. Is that correct and if so, how would the provider become associated with this instance?

Reply:
I was confused thinking it was referencing the bridge contract and not the bridge class. You can disregard comment